### PR TITLE
Fix some grammatical errors in README documentation

### PR DESCRIPTION
**Pull request by nnarct**
I corrected some of grammatical errors in your README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ class ExampleServer extends Springpress {
 }
 ```
 
-Don't forget to call `Springpress#listen` to binds and listens for connections. This method returns an [http.Server](https://nodejs.org/docs/latest-v16.x/api/http.html#class-httpserver) object (the built-in HTTP module). *You can use this instance in your testing framework to work with HTTP*.
+Don't forget to call `Springpress#listen` to bind and listen for connections. This method returns an [http.Server](https://nodejs.org/docs/latest-v16.x/api/http.html#class-httpserver) object (the built-in HTTP module). *You can use this instance in your testing framework to work with HTTP*.
```
Comment (eltfshr) [inline README.md:42]: I think we should also change `bind and listen for` to `bind and listen to`.
Otherwise, LGTM👍

Comment (riflowth) [reply README.md:42]: @nnarct What's your opinion? I can't comment anything. My English is bad. 😢

Comment (nnarct) [reply README.md:42]: @eltfshr I think _listen_ in this case means listening until the Springpress notices some connections so **for** would be the best to describe that. But I don't exactly know how the Spingpress#listen works so I could be wrong maybe.

Comment (eltfshr) [reply README.md:42]: I actually agree with you here, but the word _bind_ is there. I think maybe we change to `bind to and listen for`

Comment (nnarct) [reply README.md:42]: Exactly! That's all perfect , I think we should just go for that. Thank you

**Comment by nnarct** [comment]
I already commit the latest one ; `to bind to and to listen for`, And all should be ready to merge since I have changed only one file.

**Comment by riflowth** [comment]
Thanks to @nnarct and @eltfshr for fixing the grammatical issues in our **Springpress** documentation.

**Pull request merged**